nss-day-cohort-52/giffygram-scorpion-pilots
# display functional favorites icon, and some css/formating

#### Changes Made
1. created a functional favorites format tracking transient/permanent state 
2. merge and solved conflicts on provider.js and postlist.js 
3. added some divs for formatting 
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout branchname
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   Test app to see if permanent state is being tracked and changes saved. and rerendered upon loading 
4. View code file.
    confirm conflicts have been corrected
